server/homebrew.api.js

```diff
@@ -27,7 +27,7 @@ const getId = (req)=>{
 	return { id, googleId };
 };
 
-const getBrew = (accessType)=>{
+const getBrew = (accessType, fetchGoogle = true)=>{
```

Comment:
Can we change the phrasing here from `fetchGoogle` to `stubOnly` or `metadataOnly` or something? Somehow that seems more accurate to what we want to do here (i.e., are we retrieving a complete brew for display, or are we just looking up a few brew properties so we can update/etc.).

Then, in a later PR we can even further reduce our bandwidth usage by making `stubOnly` mode not retrieve brew text contents, which is the majority of a brew's file size. That could save us a lot of errors for users with big brews that end up timing out after repeated autosaves takes up all the traffic.